node/network/statement-distribution/src/lib.rs

```diff
@@ -269,7 +269,9 @@ impl PeerRelayParentKnowledge {
 			CompactStatement::Seconded(ref h) => {
 				self.seconded_counts.entry(fingerprint.1).or_default().note_local(h.clone());
 
-				self.sent_candidates.insert(h.clone())
+				let was_known = self.is_known_candidate(h);
+				self.sent_candidates.insert(h.clone());
+				was_known
```

Comment:
Shouldn't this be !was_known ... the function is supposed to return whether the pear learned from this statement the first time, which is equivalent to it was not known by the peer.